### PR TITLE
fix: redirect kube page to pods page

### LINKED ISSUE #8652: Done button displayed after loading of a yaml file is not clickable in the UI on main branch
### Bug description

Currently it's not possible to click the `Done` button that is displayed after the user finishes loading/playing a yaml file while running PD from current main branch.

https://github.com/user-attachments/assets/99f0b2bf-376e-43b2-b82c-d0870ad51378

### Operating system

macOS

### Installation Method

Installer from website/GitHub releases

### Version

next (development version)

### Steps to reproduce

1. Open PD from current main branch.
2. Go to Pods page.
3. Play an yaml file.
4. Try to press the `Done` button when the yaml finishes.
5. Notice that the `Done` button can't be clicked even though it reacts to mouse hover and looks enabled.

### Relevant log output

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by deboer-tim**
### What does this PR do?

window.history.go doesn't work, just goto the pods page since that is the only page the play kube button is available from and equivalent to what we do on some other forms.

### Screenshot / video of UI

N/A

### What issues does this PR fix or reference?

Fixes #8652.

### How to test this PR?

Go to Pods page and click Play Yaml, play a valid yaml file and then confirm clicking Done works.

### CODE REVIEW

File: packages/renderer/src/lib/kube/KubePlayYAML.svelte [hunk 1]
```diff
@@ -155,8 +156,9 @@ onDestroy(() => {
   }
 });
 
-function goBackToHistory(): void {
-  window.history.go(-1);
+function goBackToPodsPage(): void {
+  // redirect to the pods page
+  router.goto('/pods');
```
Comment (axel7083) [inline packages/renderer/src/lib/kube/KubePlayYAML.svelte]:
```ts
import { handleNavigation } from '/@/navigation';
import { NavigationPage } from '/@api/navigation-page';

handleNavigation({
  page: NavigationPage.PODS,
});
```

Comment (deboer-tim) [reply packages/renderer/src/lib/kube/KubePlayYAML.svelte]: Fixed.

Comment (axel7083) [reply packages/renderer/src/lib/kube/KubePlayYAML.svelte]: Missing push ?

Comment (deboer-tim) [reply packages/renderer/src/lib/kube/KubePlayYAML.svelte]: Yes, fixed now 😆

**Review by axel7083 — APPROVED** [review]
LGTM :rocket:

**Review by benoitf — CHANGES_REQUESTED** [review]
Code LGTM but missing unit tests

**Review by axel7083 — APPROVED** [review]
LGTM

**Review by benoitf — APPROVED** [review]

**Pull request merged**